apps/partners-reference/cypress/integration/navigation.ts

```diff
@@ -0,0 +1,23 @@
+describe("Navigating around the site", () => {
+  it("Loads the homepage directly", () => {
```

Comment:
What this test is really checking is that our redirect works. Let's call it out explicitly: `it("Redirects to sign-in without a logged-in user"`